lux.toml

```diff
@@ -0,0 +1,24 @@
+package = "Beyond All Reason"
+version = "0.1.0"
+lua = "=5.1"
+
+[description]
+summary = "Beyond All Reason Game Code."
+maintainer = "BAR Team"
+labels = [ "BAR" ]
+license = "GPL"
+
+[test_dependencies]
+busted = "2.2.0"
+luassert = "1.9.0"
+inspect = "3.1.3-0"
+
+[test_dependencies.luassert_intellisense]
+git = "github:LuaCATS/luassert"
+version = "d3528bb679302cbfdedefabb37064515ab95f7b9"
+luarc = "etc"
```

Comment:
Ok, putting this back into draft for a hot second while i try to figure out a way to get the busted/luassert types into the project with lux:  https://github.com/lumen-oss/lux/pull/1190

TLDR because the LuaCATS repo does not expose a rockspec, lux has no idea how to add the types it to `.luarc.json` (thereby enabling the types in the language server) and does it wrong.